### PR TITLE
update Home page to v3.1

### PR DESCRIPTION
**Pull request by toucham**
**Changes:**
- make navbar overlay photo
- update homepage to v3.1 of the designs
- support mobile

### CODE REVIEW

File: src/styles/ContentLayout.module.less [hunk 1]
```diff
@@ -2,8 +2,7 @@
 
 .contentlayoutcontainer {
   width: 100vw;
-  padding: 3% 9%;
-  padding-bottom: 6%;
+  padding: @navbar-height+3rem 9%;
```
Comment (IanSaucy) [inline src/styles/ContentLayout.module.less:5]: I think you need to use `calc(..)` for this calculation to actually work. Probably why the padding looks a bit different right now.

**Review by IanSaucy — CHANGES_REQUESTED** [review]
**Really good** work on everything. I think you just need the calc() in one part. 

Also noticed that the recent media and publications images/content should be left aligned to the title. Right now they seem to float separately from the title. I know this might cause some weird things with the mobile cutoff but that is probably fine. 

Also want to tag in @minkyoseo to give thoughts on the padding. The changes reduce the padding a bit but want to make sure it's still fine! I think the less padding is all around good but want to double check.

**Comment by Ingi-Hong** [comment]
Things I noticed: 
- CEID logo text should be white on the landing page (right now its black)
- I noticed its kind of hard to see Home navigation button when its selected, since it's that dark cyan color on a black background. Maybe @minkyoseo has some thoughts on changing the selected color? 

Awesome job :D

**Review by Ingi-Hong — COMMENTED** [review]
Things I noticed:

- CEID logo text should be white on the landing page (right now its black)
- I noticed its kind of hard to see Home navigation button when its selected, since it's that dark cyan color on a black background. Maybe @minkyoseo has some thoughts on changing the selected color?

Awesome job :D

**Comment by IanSaucy** [comment]
Chiming in about the logo, since it's all a single SVG file we will need to apply a style on the SVG or have an alternative to display that's inverted. I've not looked at the SVG to see how easy it would be to selectively style just the text and or create an inverted version. 

I think we log that as a "bug" in Trello.

**Review by IanSaucy — APPROVED** [review]
lgtm 

Log a bug for the mobile design and how the centering should be done etc. @minkyoseo Will need to chime in for that one.

**Pull request merged**